maps/away/blueriver/blueriver.dm

```diff
@@ -1,8 +1,8 @@
 //quality code theft
 #include "blueriver_areas.dm"
 /obj/effect/overmap/visitable/sector/arcticplanet
-	name = "arctic planetoid"
-	desc = "Sensor array detects an arctic planet with a small vessle on the planet's surface. Scans further indicate strange energy levels below the planet's surface."
+	name = "arctic moon"
+	desc = "Sensor arrays detect a tiny moon with sub-zero surface conditions and sparse flora, likely seeded from Earth species.<br><br>Topographical scans highlight the profile of a small vessel, located on the western side of a valley surrounded by mineral-rich mountains, but it isn't broadcasting any callsign or identification codes. Subterranean energy signatures are bizarre and constantly fluctuating, and don't at all match expected levels.<br><br>It has a very long rotational period, with sunrise in an estimated 56 hours."
```

Comment:
Since the idea (I believe) is that the sensor readout is computer-generated, it should be written less personally:
```suggestion
	desc = "Sensor arrays detect a tiny moon with sub-zero surface conditions and sparse flora, potentially seeded from Earth species.<br><br>Topographical scans highlight the profile of a small vessel located on the western side of a valley surrounded by mineral-rich mountains with no broadcasting transponder. Subterranean energy signatures are present and constantly fluctuating, in excess of expected levels.<br><br>The moon has a very long rotational period, with sunrise in an estimated 56 hours."
```